common/prettify.pm

```diff
@@ -670,6 +671,28 @@ sub Description ($)
 
 sub Timestamp ($)
```

Comment:
```suggestion
sub Timestamp
```

Reply:
What does this do? I'm hesitant to change this line since I think it's meant to match the other Timestamp function signatures?

Reply:
According to what I could find in the Perl documentation and the only significant OO Perl codebase I'm familiar with (MPC) we should not use function prototypes here.